README.md

```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
    - A robust foundation model, trained on 117 classes encompassing various organs and tumors.
    - The model's training and validation involved CT images from over 4,000 patients, ensuring its high accuracy and reliability for medical image analysis.
 
-### How to Get Started
+4. Dicon-web Setup and Dataset Configuration:
+   - [Dicom-web Overview](#dicomweb-overview)
+   - [Notebook: Dataset Creation and Experiment Selection](https://github.com/NVIDIA/monai-cloud-api/blob/main/notebooks/Dataset%20Creation%20and%20Experiment%20Selection.ipynb)
+
+5. OHIF Plugin and Overview:
+   - [Getting Started with OHIF Plugin](#getting-started-with-ohif-plugin)
+   - [Setup Customized OHIF ](https://github.com/NVIDIA/monai-cloud-api/tree/main/plugins/ohif)
+   - [OHIF Applications and UI Demo](#basic-ohif-instruction)
+   - [Interactive Annotation](#interactive-annotation)
+   - [Notebook: Annotation and OHIF Overview](https://github.com/NVIDIA/monai-cloud-api/blob/main/notebooks/Annotation%20and%20Continuous%20Learning%20Overview.ipynb)
+
+
+## How to Get Started
 
 To begin using the NVIDIA MONAI Cloud APIs, you can sign up for early access by filling out this survey: [NVIDIA MONAI Cloud API Early Access](https://developer.nvidia.com/nvidia-monai-cloud-api-early-access-program/join). 
+
+## Getting Started with OHIF Plugin
+
+MONAI Service provide ready-to-use playgroundw with API server and applications for users. 
+If you are going to build and custimize OHIF viewer and MONAI Service plugin. Please refer to the [setup page](https://github.com/NVIDIA/monai-cloud-api/tree/main/plugins/ohif)
+
+### Dicomweb Overview
+Dataset storage can be variant, in the open-sourced setup. We recommend Orthanc for hosting dicom format medical images. 
```

Comment:
The storage of datasets can vary; in our open-source setup, we recommend using Orthanc for hosting **DICOM** format medical images.